### PR TITLE
Handle errors on url

### PR DESCRIPTION
**Pull request by enguerran**
If a user write a non `https://` url, an error is displayed.

If a user write only his domain name, `https://` is added.

What do you think ?

PS : I refactor i18n keys for homogeneity.

### CODE REVIEW

**Comment by enguerran** [comment]
I wait for a solution to disable `https://` check when we are in development mode.

**Comment by m4dz** [comment]
> I wait for a solution to disable https:// check when we are in development mode.

Woohoo 🙌 ! Solution incoming in #56 🌮

**Comment by enguerran** [comment]
@kosssi I commit this https://github.com/cozy/cozy-files-v3/pull/52/commits/9c300bb9c34eca595982213602dd208a0bba1c8e to let this PR be merged before #56. What do you think? May I remove **[WIP]** from this PR's title and let you merge it?

**Comment by m4dz** [comment]
@enguerran @kosssi what we do w/ this one? Do you need to rework on it?

**Comment by enguerran** [comment]
Ok, I'll rebase this one on top of master and update the failing test.

**Comment by enguerran** [comment]
@m4dz could you review what I did for tests? https://github.com/cozy/cozy-files-v3/pull/52/commits/73a54368262919fa32b4007d91becb16475dd5a6

@kosssi could you review everything else?

File: mobile/src/containers/OnBoarding.jsx [hunk 1]
```diff
@@ -10,15 +10,16 @@ import { registerDevice, SET_URL } from '../actions'
...
-export const SelectServer = ({selectServer, t, updateServerUrl, serverUrl}) =>
+export const SelectServer = ({selectServer, t, updateServerUrl, serverUrl, error}) =>
 (
   <div className={classnames(styles['wizard'])}>
     <div className={classnames(styles['wizard-main'])}>
-      <p>{t('mobile.wizard.selectServer.cozy_address')}</p>
-      <input type='text' placeholder={t('mobile.wizard.selectServer.cozy_address_placeholder')} onChange={updateServerUrl} value={serverUrl} />
-      <p>{t('mobile.wizard.selectServer.description')}</p>
+      <p>{t('mobile.onboarding.serverselection.cozy_address')}</p>
+      <input className={styles['http-input']} type='url' pattern='https://.+' placeholder={t('mobile.onboarding.serverselection.cozy_address_placeholder')} onChange={updateServerUrl} value={serverUrl} />
```
Comment (kosssi) [inline mobile/src/containers/OnBoarding.jsx]: You have `pattern='https://.+'`, it's not good when you are a on development?

File: src/locales/en.json [hunk 2]
```diff
@@ -60,16 +60,17 @@
...
   },
   "mobile": {
-    "wizard": {
+    "onboarding": {
       "welcome": {
         "title": "Your own private cloud",
         "button": "Sign in"
       },
-      "selectServer": {
+      "serverselection": {
```
Comment (kosssi) [inline src/locales/en.json]: Can you refactoring 'serverselection' to 'server_selection'?

**Pull request merged**